wayfire.desktop.in

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Version=0.4
+Name=Wayfire Session
+Comment=Use this session to run Wayfire as your desktop environment
+Exec=$PREFIX/bin/startwayfire
+Icon=$PREFIX/share/wayfire/icons/wayfire.png
```

Comment:
The icon here is installed by wcm but maybe wayfire should install it instead?

Reply:
Yes, that would make sense. I didn't realize this inconsistency.